ca_tools/launch/keyboard_teleop.launch

```diff
@@ -1,3 +1,5 @@
+<!-- USE MOVE_ROBOT VARIABLE TO SELECT WHICH ROBOT TO CONTROL, DEFAULT=1 -->
 <launch>
-  <node pkg="ca_tools" type="key_teleop.py" name="keyboard_teleop" output="screen"/>
+  <arg name="nr" default="$(optenv MOVE_ROBOT 1)"/>
```

Comment:
I would rename `MOVE_ROBOT` to `ID`. Also the argument `nr`.

Reply:
Adressed